api/src/main/java/org/apache/iceberg/util/CharSequenceWrapper.java

```diff
@@ -81,4 +81,9 @@ public char charAt(int index) {
   public CharSequence subSequence(int start, int end) {
     return wrapped.subSequence(start, end);
   }
+
+  @Override
+  public String toString() {
+    return wrapped.toString();
```

Comment:
This bug was flagged by my IDE. CharSequence requires overriding `toString`, but the implementation was provided by `Object`.